test/e2e/helpers.js

```diff
@@ -741,36 +741,16 @@ const sendScreenToConfirmScreen = async (
   await openActionMenuAndStartSendFlow(driver);
   await driver.fill('[data-testid="ens-input"]', recipientAddress);
   await driver.fill('.unit-input__input', quantity);
-  if (process.env.MULTICHAIN) {
```

Comment:
this logic is outdated, making specs in multichain fail.

After evaluating the failure, it is not a straight-forward fix and there is some complexity (the wallet freezes and being able to get mock responses - possibly need to check this with the Multichain team, might be an issue on their end too). For this reason, this fix is out of scope of this PR and should be handled separately.

Note: this method was currently only used by PPOM tests, so no other spec is affected